server/src/main/java/org/elasticsearch/index/mapper/CompletionFieldMapper.java

```diff
@@ -411,13 +422,21 @@ private void checkCompletionContextsLimit(BuilderContext context) {
                 }
             }
         }
+
+        @Override
+        public Builder index(boolean index) {
+            if (index == false) {
+                throw new MapperParsingException("Completion field type must be indexed");
```

Comment:
new error?

Reply:
It was ignored before, should be a warning in 7x